.circleci/config.yml

```diff
@@ -72,6 +100,64 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  e2e-simple-cni:
+    <<: *integrationDefaults
+    environment:
+            - KUBECONFIG: /go/out/minikube.conf
+            - TEST_ENV: minikube-none
+            - GOPATH: /go
+    steps:
+      - <<: *initWorkingDir
+      - checkout
+      - attach_workspace:
+          at:  /go
+      - <<: *markJobStartsOnGCS
+      - run: make sync
+      - run: make build
+      - run:
+          command: |
+            if [ ! -f /go/out/linux_amd64/release/istio-cni ]; then
+              # Should only happen when re-running a job, and the workspace is gone
+              time make build
+            fi
+            make docker.all
+      - run: docker images
+      - run:
+          name: Get Istio
+          shell: /bin/bash
+          working_directory: /go/src/istio.io/istio
+          command: |
+            git clone https://github.com/tiswanso/istio.git .
+            git checkout istio-cni_helm
+      - run: /go/src/istio.io/istio/bin/testEnvRootMinikube.sh start
+      - run: /go/src/istio.io/istio/bin/testEnvRootMinikube.sh wait
+      - run: docker pull gcr.io/istio-release/galley:master-latest-daily
+      - run: docker images
+      - run:
+          name: Apply Istio CRDs
+          shell: /bin/bash
+          working_directory: /go/src/istio.io/istio
+          command: |
+            kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
+      - run:
+          name: Run Istio e2e_simple
+          shell: /bin/bash
+          working_directory: /go/src/istio.io/istio
+          command: |
+            HUB=gcr.io/istio-release TAG=master-latest-daily make istioctl
+            HUB=gcr.io/istio-release TAG=master-latest-daily ENABLE_ISTIO_CNI=true EXTRA_HELM_SETTINGS="--set istio-cni.excludeNamespaces={} --set istio-cni.tag=v0.1-dev" E2E_ARGS="--kube_inject_configmap=istio-sidecar-injector" make test/local/auth/e2e_simple
```

Comment:
Interesting paradox here.  Should change the v0.10dev label to match the label corresponding to this PR.   Again since the test doesn't pass maybe it's not critical

Reply:
the test should pass with these changes.  It passes locally.  But yeah, we need to move the tag to the build that's happening for each PRs diffs.